Source/ViewControllers/BrickCollectionView.swift

```diff
@@ -501,7 +509,10 @@ extension BrickCollectionView: AsynchronousResizableDelegate {
         weak var weakLayout = self.layout
         self.performBatchUpdates({
             weakLayout?.updateHeight(indexPath, newHeight: newHeight)
-            }, completion: completion)
+        }, completion: { completed in
+            let _ = self
```

Comment:
I am note sure. I was seeing the issue in invalidateVisibility. I can test it out real quick.